### PR TITLE
[docs] Add missing apt-get update

### PR DESCRIPTION
**Pull request by kamranayub**
I was following the steps and couldn't run the apt commands until I re-ran `sudo apt-get update`.

Also, I tried copy/pasting step 1. into my terminal but it wasn't formatted correctly on paste. Easy fix but threw me for a second.

image

### CODE REVIEW

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/install-azure-cli-apt.md](https://github.com/kamranayub/azure-docs-cli-python/blob/patch-1/docs-ref-conceptual/install-azure-cli-apt.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/install-azure-cli-apt?view=azure-cli-latest&branch=pr-en-us-1028) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/install-azure-cli-apt?view=azure-cli-2017-03-09-profile&branch=pr-en-us-1028) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C14%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201809140338255878-1028%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=h1NFE%2B368SCsT4ZINk6vrDMp32VMf3oarzuYDK%2BEOxg%3D&st=2018-09-14T03%3A38%3A25Z&se=2018-10-15T03%3A43%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/install-azure-cli-apt.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ for the Azure CLI. This package has been tested with:
 3. Install the CLI:
 
    ```bash
+   sudo apt-get update
    sudo apt-get install apt-transport-https
    sudo apt-get update && sudo apt-get install azure-cli
```
Comment (sptramer) [inline docs-ref-conceptual/install-azure-cli-apt.md]: Since `apt-get update` is now earlier in the script, it can be removed here.

**Comment by sptramer** [comment]
Thanks for your contribution! Please address feedback and this can be merged.

File: docs-ref-conceptual/install-azure-cli-apt.md (hunk 1 above)
Comment (kamranayub) [reply docs-ref-conceptual/install-azure-cli-apt.md]: Thanks, also I noticed `apt-get install` can now be collapsed

Comment (kamranayub) [reply docs-ref-conceptual/install-azure-cli-apt.md]: Do you think after installing `apt-transport-https` doing an update might still be necessary? I guess I'm not sure...

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/install-azure-cli-apt.md](https://github.com/kamranayub/azure-docs-cli-python/blob/patch-1/docs-ref-conceptual/install-azure-cli-apt.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/install-azure-cli-apt?view=azure-cli-latest&branch=pr-en-us-1028) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/install-azure-cli-apt?view=azure-cli-2017-03-09-profile&branch=pr-en-us-1028) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C9%5C14%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201809141756355768-1028%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=35Q8IdaIjljwuLtwjXLDpPlZSfWoMQ3Y8%2BArXc9xX3Q%3D&st=2018-09-14T17%3A55%3A42Z&se=2018-10-15T18%3A00%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**